tests/builders/test_tools.py

```diff
@@ -44,6 +45,37 @@
 from bluemira.geometry.wire import BluemiraWire
 
 
+class TestGetNSectors:
+    sector_degree = [
+        360.0,
+        180.0,
+        120.0,
+        90.0,
+        72.0,
+        60.0,
+        51.42857142857143,
+        45.0,
+        40.0,
+        36.0,
+        32.72727272727273,
+        30.0,
+        27.692307692307693,
+        25.714285714285715,
+        24.0,
+    ]
+
+    @pytest.mark.parametrize("ttl, sector_degree", zip(np.arange(1, 16), sector_degree))
+    @pytest.mark.parametrize("degree", np.arange(0, 361, step=60))
+    def test_get_n_sectors(self, degree, ttl, sector_degree):
+        s_deg, n_sec = get_n_sectors(ttl, degree)
+        assert np.isclose(s_deg, sector_degree)
+        try:
+            assert np.isclose(degree // sector_degree, n_sec)
+        except AssertionError:
+            # a sector at both ends
+            assert np.isclose((degree // sector_degree) + 1, n_sec)
```

Comment:
Is this basically doing the same calculations as the function it's testing? Could we consider just calculating a few examples by hand and checking those? I think that's the commonly done thing, rather than duplicating the logic under test. Not a big deal though

Reply:
Wrong PR :) its isolated in #1327. I was fuzzing it a bit but yeah I can do that

Reply:
done in [2cc6a60](https://github.com/Fusion-Power-Plant-Framework/bluemira/pull/1327/commits/2cc6a603b999176dcaba7a06c12ed8cd0464a181)